test/e2e/operator_test.go

```diff
@@ -368,9 +367,6 @@ func customResolver(nameserver string) *net.Resolver {
 // with multiple ingress controller deployed in Openshift.
 // Route's host should resolve to the canonical name of the specified ingress controller.
 func TestExternalDNSCustomIngress(t *testing.T) {
-	if operandVersion == "v0.10.1" {
```

Comment:
Let's make this PR WIP for the moment.
A pre-release of `0.10.2` was done last week, so hopefully it'll come out soon.